crates/wasmtime/src/config.rs

```diff
@@ -36,6 +36,7 @@ pub use wasmtime_environ::CacheStore;
 
 /// Represents the module instance allocation strategy to use.
 #[derive(Clone)]
+#[non_exhaustive]
```

Comment:
Not sure I get why this must now be non-exhaustive?

Reply:
Ah this should have been non_exhaustive before because one variant is conditionally defined, so I figured I'd go ahead and fix it while I was here